unison-link.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+ln -s /bin/unison-fsmonitor-$UNISON_VERSION /bin/unison-fsmonitor
```

Comment:
Oh right, it needs to happen each time, because the ENV `UNISON_VERSION` could be changed between runs. :smile: